### PR TITLE
Game Rules

### PR DESCRIPTION
**Pull request by JorgeSedek**
## Motivation
i consider that description of the rules is finished and could be added to the main.
Closes #

## Summary of changes
added the image of card values, description of how to play and what is envido, flor and some minor chances to the readMe.
## Checklist
- [ ] Tested the changes locally.
- [ ] Reviewed the changes on GitHub, line by line.
- [ ] Unit tests added.
- [x] This change requires new documentation.
  - [x] Documentation has been added/updated.

### CODE REVIEW

**Comment by maxogod** [comment]
ya habia un RULES.md en la carpeta docs que esta conectado al README.md y cuando en el README apretas "Game Rules" te lleva al viejo RULES.md que no tiene contenido, podrias copiar lo del nuevo RULES.md al otro? :)

**Comment by maxogod** [comment]
Tambien en el RULES.md hay titulo para "Envido" y "Flor" pero no hay titulo para "Truco", y tampoco se explica bien como jugar al envido por ejemplo o como contar los puntos para la contraflor. Cuando agregues eso mergeamos, pero igual el formato esta bueno!😁

**Comment by MelnikAndres** [comment]
.

File: docs/RULES.md [hunk 1]
```diff
@@ -1,5 +1,73 @@
...
+
+   * **Quiero**: accepts to take the bet.
+   * **Envido**: raise the bet by 2 points.
+   * **Real envido**: raise the bet by 3 points.
+   * **Falta envido**: if accepted the one who wins, wins the game.
+   * **No quiero** - refuses to take the last bet. 
+
+If no one calls for envido, there is no score for it.   
+The cards are counted as follow:
+* same suit: the sum of the two cards + 20.
```
[the remark this answers is not in the document]
Comment (MelnikAndres) [reply docs/RULES.md:44]: .

File: docs/RULES.md [hunk 2]
```diff
@@ -1,5 +1,73 @@
...
+
+![trucoCards](https://github.com/maxogod/Truco/assets/85034225/627d81df-2c5d-404a-9927-695783b62e58)
+
+### Sources
+
+1. [wikipedia truco](https://en.wikipedia.org/wiki/Truco)
+1. [argentinian truco rules](https://www.pagat.com/put/truco_ar.html)
+1. [description truco in city Buenos Aires page](https://turismo.buenosaires.gob.ar/en/article/get-tricks-%E2%80%9Ctruco%E2%80%9D)
+1. [rules truco in a online game](https://www.mundigames.com/multiplayer/argentinian-truco/rules/)
+
```
Comment (maxogod) [inline docs/RULES.md:70]: just a test of the review functionality

File: docs/RULES.md [hunk 3]
```diff
@@ -1,5 +1,92 @@
...
+        envido (player 1) -> envido (player 2) -> real envido (player 1) -> quiero (player 2)
+        
+The player who has the best pair will win 7 points (2 + 2 + 3).
+
+Another example with _No quiero_ could be:
+
+        envido (player 1) -> envido (player 2) -> real envido (player 1) -> real envido (player 2) -> falta envido (player 1) -> no quiero (player 2)
+        
+player 1 wins 10 points (2 + 2 + 3 + 3).
```
Comment (maxogod) [inline docs/RULES.md:57]: When a player rejects a bet with "no quiero" an additional point goes to the one who raised the bet (+1). Its specified bellow this line but the additional point is not accounted for in this point calculation

**Review by AndreasK24 — APPROVED** [review]

**Pull request merged**